cacheops/conf.py

```diff
@@ -38,7 +39,7 @@ def _setup(self):
         except AttributeError:
             raise ImproperlyConfigured('You must specify CACHEOPS_REDIS setting to use cacheops')
 
-        client = (SafeRedis if DEGRADE_ON_FAILURE else redis.StrictRedis)(**redis_conf)
+        client = (SafeRedis if DEGRADE_ON_FAILURE else LocalCachedTransactionRedis)(**redis_conf)
```

Comment:
Probably move all redis class to `cacheops/redis.py` and import from where. `conf.py` will become data only, which is good.